docs/development/autoscaling-specifics-for-components.md

```diff
@@ -36,57 +36,17 @@ For both of the autoscaling modes downscaling is handled more pessimistically to
 
 ## Shoot Kubernetes API Server
```

Comment:
https://github.com/plkokanov/gardener/blob/229689f7fc5cd3568318ba492839d9c01d275099/docs/development/autoscaling-specifics-for-components.md?plain=1#L59-L61 needs to be adapted as autoscaling modes are removed with this PR.